.github/workflows/release.yml

```diff
@@ -77,6 +77,11 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      -
+        name: Set up yq
+        run: |
+          sudo wget https://github.com/mikefarah/yq/releases/download/v4.7.0/yq_linux_amd64 -O /usr/bin/yq
```

Comment:
NOOOO! this is the wrong yq.

Reply:
I think you want https://github.com/eclipse/che-release/blob/master/.github/workflows/release-orchestrate-overall.yml#L72-L78